### DISCUSSION TITLE
iframe iframeMaxWidth in v2

### DISCUSSION THREAD

**Discussion by apload-gmbh** [sachinchoolur/lightGallery#1180] [Q&A]
Hi

We are planning to migrate from Lg 1.2.6 to 2.1.1 and running some tests.
We used to display iframe popups the attribute iframeMaxWidth, but it seems tha only iframeWidth exists on v2. 
In our opinion max-with (in CSS) would suit better for responsive behavior on smartphones.
Is there still an option the get that?

Thanks, Jan

**Comment by sachinchoolur** [comment]
Hey @apload-gmbh,
Really sorry for the late reply. I always forget to check the discussions.

Do you have any scenario where width is not sufficient? 
It is very easy to add maxWidth if required

**Reply by apload-gmbh** [reply]
Thanks for your answer

The content in the iframe sometimes needs a certain width (or height) and percents aren't a good solution because on FHD screens 50% is great, but on mobile devices 50% would cut off too much information. So mobile 100% width would be better.
A combination of max-width and width (max-height and height) would let us use max-width: 100% and width: 700px. In such a case, the iframe could not overlap the screen and would allow to scroll in the iframehorizontally.

In the old version width was set 100% (maybe we changed ourselves your css file locally) and only max-width (iframeMaxWidth) could be set as an attribut.

Unfortunately it's difficult to show you examples, because the relevant part of it is in the backend of our cms.

Do you understand why max-width and max-height are very helpful for this kind of iframe-szenarios or do I have to prepaire a Codepen?

Regards, Jan

**Reply by sachinchoolur** [reply]
Thank you for the detailed explanation.
Added `iframeMaxWidth` and `iframeMaxHeight` options in version [2.3.0](https://github.com/sachinchoolur/lightGallery/releases/tag/2.3.0)
